server/routes/api/place.py

```diff
@@ -68,7 +93,7 @@ def name():
     for dcid in dcids:
         values = response[dcid].get('out')
         result[dcid] = values[0]['value'] if values else ''
-    return json.dumps(result)
+    return Response(json.dumps(result), 200, mimetype='application/json')
```

Comment:
curious what is the difference with the bare json.dump

Reply:
the browser recognizes the content as json, and displays nice formatting and folding features :) it was helpful for debugging.

Reply:
nice, can gradually move the rest of the reponse